### ISSUE TITLE
[Github] Add heading elements for account login page to improve accessibility & SEO

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5457]
### 1. Why is this change necessary?
This change is necessary to enhance the accessibility of the account login page. Proper use of heading tags is a key principle of accessible web design as outlined by the Web Content Accessibility Guidelines (WCAG). Not only do they help screen reader users understand the structure and navigate the content more effectively, but they also provide semantic meaning to the document for assistive technologies. Additionally, although SEO is not a primary concern for a login/signup page, having a well-structured document is a good practice and ensures consistency across the platform.

### 2. What does this change do, exactly?
The change introduces h2 heading tags to the login-card and register-card sections of the account login page. Ideally you would also want an h1 element on the page, but this currently doesn't exist and I'd say the login-card and register-card are equal to each other in importance.

### 3. Describe each step to reproduce the issue or behaviour.
- Go to the account login page e.g. https://shopware.bluebirdday.io/account/login 
- Inspect the login-card & register-card title
- They are not wrapped in a heading element

### 5. Checklist

- [x] I have rebased my changes to remove merge conflicts
- [ ] I have written tests and verified that they fail without my change
- [ ] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-implement-new-changelog.md) with all necessary information about my changes
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [ ] I have read the contribution requirements and fulfil them.

---

Imported from Github. Please see: https://github.com/shopware/shopware/pull/3659

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-35318**



[created from NEXT-35318, comment 511883]
